src/transaction/log_recovery_redo.hpp

```diff
@@ -644,6 +645,17 @@ void log_rv_redo_record_sync_apply (THREAD_ENTRY *thread_p, log_rv_redo_context
 			 VPID_AS_ARGS (&rcv_vpid), (int) rcv.offset, (int) rcv.length);
     }
 
+  if (is_passive_transaction_server() && log_data.rcvindex == RVPGBUF_DEALLOC)
+    {
+      /* TODO:
+       * redofunc for RVPGBUF_DEALLOC (pgbuf_rv_dealloc_redo ()) invalidates the page buffer in PTS,
+       * but it does not set page lsa (pgbuf_set_lsa ()) before invalidation.
+       * We need to set page lsa when redoing RVPGBUF_DEALLOC log, as what ATS does (see the caller of LOG_NEED_TO_SET_LSA ()).
+       */
+      assert (rcv.pgptr != nullptr && pgbuf_get_fix_count (rcv.pgptr) <= 0);
```

Comment:
Why `pgbuf_get_fix_count (rcv.pgptr) <= 0` and not `pgbuf_get_fix_count (rcv.pgptr) == 0` ?